components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -24,9 +24,9 @@ interface StartWorkspaceModalProps {
 type Mode = 'Recent' | 'Examples';
 
 function Tab(p: { name: Mode, selection: Mode, setSelection: (selection: Mode) => any }) {
-    const selectedTab = 'border-gray-800 text-gray-800';
+    const selectedTab = 'border-gray-800 dark:border-gray-200 text-gray-800 dark:text-gray-200';
     const inactiveTab = 'border-none text-gray-400';
-    return <div onClick={() => p.setSelection(p.name)} className={"cursor-pointer py-2 px-4 border-b-4 " + (p.selection === p.name ? selectedTab : inactiveTab)}>{p.name}</div>
+    return <div onClick={() => p.setSelection(p.name)} className={"cursor-pointer py-2 px-4 border-b-4 border-gray-200 dark:border-gray-700 " + (p.selection === p.name ? selectedTab : inactiveTab)}>{p.name}</div>
```

Comment:
**issue**: Bottom border color should be `gray-400` and on hover `gray-600`. Let's see how this looks!

Reply:
@gtsiolis That's just for dark theme, right? Light theme stays `gray-200` always?

Reply:
Aha! First, there's currently an duplicate border color there (see `border-gray-800` in rendering) that overrides `border-gray-200`. Light these could use `gray-800` and `gray-600` accordingly. We didn't have a hover style defined before now.